video/out/vo.h

```diff
@@ -202,6 +202,8 @@ enum {
     VO_CAP_NORETAIN     = 1 << 2,
     // VO supports applying film grain
     VO_CAP_FILM_GRAIN   = 1 << 3,
+    // VO is drawn untimed and is never redrawn
+    VO_CAP_UNTIMED      = 1 << 4,
```

Comment:
> There's a weird mix and match of ways to set special things about a VO so let's consolidate it all to use VO_CAP bits.

I think it's because it was VO "capabilities" and the rest was properties. It's fine to have it in one bitmap.